spec/core_functions/math/log.hrx

```diff
@@ -0,0 +1,162 @@
+<===> options.yml
+---
+:todo:
+- sass/libsass#3044
+
+<===>
+================================================================================
+<===> input.scss
```

Comment:
A test file shouldn't have subdirectories that also contain tests (this isn't in the style guide but it probably should be), so this should be in its own directory. I'd call it `positive` to match the other specs here.

Reply:
Done.